### PR TITLE
[emacs-lisp] Added emr integration

### PR DESCRIPTION
**Pull request by yyoncho**
see https://github.com/Wilfred/emacs-refactor

- I got my inspiration for the keybindings from the clojure layer

### CODE REVIEW

File: layers/+lang/emacs-lisp/README.org [hunk 1]
```diff
@@ -229,3 +230,22 @@ In =debugger-mode= (=Debugger= is displayed in major mode segment of the mode
 line)
 
 TODO
+
+** Refactoring with emr
+
+| Key binding | Description                  |
+|-------------+------------------------------|
+| ~SPC r f e~ | implement function           |
```
Comment (sdwolfz) [inline layers/+lang/emacs-lisp/README.org:238]: Since these are major mode specific, they go under `SPC m r` and not `SPC r`

Comment (yyoncho) [reply layers/+lang/emacs-lisp/README.org]: Fixed here and in changelog, thanks.

**Comment by smile13241324** [comment]
Woa thats a great addition to the elisp layer, I wonder whether something similiar could be used for the other layers too ...

**Comment by yyoncho** [comment]
> I wonder whether something similiar could be used for the other layers too ...

I guess here we should depend on what lsp servers could provide...

**Comment by smile13241324** [comment]
Jep you are right :)

**Pull request merged**